Mention Arch Linux package on README

kazuoteramoto opened the pull request:
I added a `stig` packages to AUR, the user maintained repository of packages of
Arch Linux. I added both a package that will track tagged releases (`stig`) and
one that follow `master` (`stig-git`) this last one depends on the git version
of `urwidtrees` for now.

I added a mention of this packages on the README, this pull request, I choose
to make it simple to not clutter the install section, most Arch Linux user are
familiar with AUR.

I branched from `dev`, should I branch from `master`?

rndusr commented:
Thanks for packaging.

But are you sure you want stig-git to follow master? Because master is
supposed to be mostly identical to the newest release on pypi. Development
happens in the dev branch.

Also, there is no need to depend on urwidtrees-git in stig-git. The only
reason I bumped the dependency is because urwidtree's setup.py crashed unless
urwid was already installed. That bug is resolved in 1.0.3.dev0 or the git
master. As long as it installs successfully, any version >=1.0.1.1 (I think)
should be fine.

kazuoteramoto commented:
> Development happens in the dev branch. 

OK, I'll change to follow dev.

> there is no need to depend on urwidtrees-git

The problem is the last release of urwidtrees is 1.0.2 so I get
```
pkg_resources.DistributionNotFound: The 'urwidtrees>=1.0.3dev0' distribution was not found and is required by stig
```

When I try to run it.

rndusr commented:
That's weird. https://pypi.python.org/pypi/urwidtrees lists
urwidtrees-1.0.3.dev0.tar.gz as the only release. And I'm pretty sure I tested
it with `pip3` when I bumped the version. Maybe this is something specific to
Arch?

But it doesn't really matter. Stick with the urwidtrees-git if that works.

rndusr commented:
@kazuoteramoto, I'm moving development from the dev branch to master. Can you change the stig-git package on AUR to use master?

I've also noticed that my choice to use the xdg module instead of pyxdg forced you to add a patch. The new release uses pyxdg and you should be able to remove the patch. (If I make your life harder than it needs to be in the future, feel free to open an issue!)

kazuoteramoto commented:
@rndusr Thanks for the heads-up I updated the stig-git package to follow master and removed the patch.

Sorry for not contacting you before I wanted to write a proper pull request/issue but I forgot.

rndusr commented:
Great, thanks.
